CliApplication.cpp

```diff
@@ -30,7 +30,7 @@ CliApplication::CliApplication( int &argc, char **argv, const QString &appName )
 }
 
 CliApplication::CliApplication( int &argc, char **argv, const QString &appName, const QString &outFile )
-	: QObject( 0 ), argc(argc), argv(argv), appName(appName), outFile(outFile)
+	: QObject(), argc(argc), argv(argv), appName(appName), outFile(outFile)
```

Comment:
I'm still not sure what is the benefit of calling the QObject constructor here (with or without parameters). I'm pretty sure we could do without it...